ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -374,6 +374,26 @@ tests:
       IP_STACK: v4
       NUM_EXTRA_WORKERS: "1"
     workflow: assisted-baremetal-operator-ztp
+- as: e2e-metal-assisted-operator-ztp-sno-with-day2-workers
```

Comment:
Do not change existing jobs.

Most jobs relating to single-node across the OpenShift organization use `single-node`

Reply:
Actually found another problem, if we use `single-node` instead of `sno` it breaks a validation on the length of the job name. I'm open to suggestions about full job name that would include `single-node`? while not breaking this?